apps/tx-builder/src/components/Dashboard.tsx

```diff
@@ -81,21 +81,18 @@ const Dashboard = (): ReactElement => {
     };
 
     loadContract(addressOrAbi);
-  }, [addressOrAbi, interfaceRepo, web3]);
+  }, [addressOrAbi, interfaceRepo]);
 
   const getAddressFromDomain = (name: string): Promise<string> => {
-    return services?.web3?.eth.ens.getAddress(name) || new Promise((resolve) => resolve(name));
+    return web3?.eth.ens.getAddress(name) || new Promise((resolve) => resolve(name));
   };
 
-  const isValidAddress = useCallback(
-    (address: string | null) => {
-      if (!address) {
-        return false;
-      }
-      return web3?.utils.isAddress(address);
-    },
-    [web3],
-  );
+  const isValidAddress = useCallback((address: string | null) => {
```

Comment:
the same function exists in `Builder.tsx`

Reply:
done [here](https://github.com/gnosis/safe-react-apps/pull/262#discussion_r785927409)! :D